tests/sdk/nodejs/examples/helm-release-prometheus/index.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright 2016-2021, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as pulumi from "@pulumi/pulumi";
+import * as k8s from "@pulumi/kubernetes";
+
+
+const namespace = new k8s.core.v1.Namespace("release-ns");
+
+const prometheusOperator = new k8s.helm.v3.Release("prometheus", {
```

Comment:
Wasn't prometheus causing races between tests with the CRD?

Reply:
> Wasn't prometheus causing races between tests with the CRD?

Good point. I remember that was prometheus-stack but we definitely have prometheus-operator in the tests as well. The test passed but I don't trust it wont race. I will find a different unique chart instead. Thanks for the flag.

Reply:
Updated to alertmanager. From what I can see that should be safe.